kornia/augmentation/utils.py

```diff
@@ -87,6 +77,10 @@ def _adapted_uniform(shape: Union[Tuple, torch.Size], low, high, same_on_batch=F
     If same_on_batch is True, all values generated will be exactly same given a batch_size (shape[0]).
     By default, same_on_batch is set to False.
     """
+    if not isinstance(low, torch.Tensor):
+        low = torch.tensor(low).float()
```

Comment:
It is actually because the ```Uniform(low, high)``` function required the low and high to be float or double. An error will be raised if passing integer or long. So I think it is better to keep it this way for now.